tests/ttnn/unit_tests/operations/test_composite.py

```diff
@@ -764,3 +764,28 @@ def test_unary_celu(input_shapes, param, device):
 
     comp_pass = compare_pcc([output_tensor], [golden_tensor])
     assert comp_pass
+
+
+@skip_for_grayskull()
+@pytest.mark.parametrize(
+    "input_shapes",
+    (
+        (torch.Size([1, 1, 32, 32])),
+        (torch.Size([1, 1, 320, 384])),
+        (torch.Size([1, 3, 320, 384])),
+    ),
+)
+@pytest.mark.parametrize(
+    "param",
+    {random.uniform(-100, 100) for _ in range(5)},
```

Comment:
should this range be changed? 
in description it says  "denominator value which is actually calculated as numerator float value >= 0",

Reply:
yes @KalaivaniMCW. Will change it.